### DISCUSSION TITLE
Query - Does Lottie Support Mac Catalyst

### DISCUSSION THREAD

**Discussion by welsh11** [airbnb/lottie-ios#1955] [Q&A]
Hi there,
I've probably messed up somewehere, but I have `lottie-spm` 4.1.2 installed as a package in my project.

I have just enabled Mac Catalyst in my project and I get a build error when I try to run the project on a mac

/<redacted>/SourcePackages/artifacts/lottie-spm/Lottie.xcframework:1:1 While building for Mac Catalyst, no library for this platform was found in '/<redacted>/Library/Developer/Xcode/DerivedData/xxx/SourcePackages/artifacts/lottie-spm/Lottie.xcframework'.

Any ideas on this? 

Thanks

**Comment by calda** [comment]
This usually means Lottie isn't embedded in the final app product. Make sure Lottie is listed under your target's "Frameworks, Libraries, and Embedded Content":

<img width="568" alt="image" src="https://user-images.githubusercontent.com/1811727/219055329-866f9fed-8098-4f1a-8717-59eecc385ed8.png">

**Reply by welsh11** [reply]
Hey there, yeah it's there and embedded.

![CleanShot 2023-02-15 at 15 48 28](https://user-images.githubusercontent.com/240153/219078721-c8f797fc-8704-4455-9a5e-dc66bfeec15d.png)

**Reply by calda** [reply]
Can you send a sample project that reproduces this issue then? If so I could take a look

**Reply by welsh11** [reply]
Hey there, here is a repro: https://github.com/welsh11/LottieCatalystTest

It's just a vanilla Swift/SwiftUI iOS app with the Mac Catalyst target added.

Thanks for taking a look!

**Reply by calda** [reply]
Thanks for the repro case! I see now that the issue is that our precompiled `xcframework` doesn't support Mac Catalyst somehow -- I didn't realize this required a different setup than just supporting normal macOS. I'll make sure to fix this. Could you file an issue on this repo to keep track of this?

**Reply by welsh11** [reply]
Done: https://github.com/airbnb/lottie-ios/issues/1958

Thanks so much for looking at this Cal.

**Reply by calda** [reply]
Thanks, this should be fixed in 4.1.3
